v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
     this.event = event
   }
 
+  /**
+   * Runs the job.
+   * 
+   * When you run the job, Brigade runs all the containers, primary and sidecar.
+   * When the primary container exits, the job ends. If sidecars are still running
+   * at this point, Brigade terminates them after a short delay.
+   * 
+   * @returns A Promise which completes when the primary container completes. If the
+   * primary container succeeded (exited with code 0), the Promise resolves; if the
+   * primary container failed (exited with nonzero code) or timed out, the Promise
+   * rejects. Sidecars do not affect success or failure of the job.
+   */
   public run(): Promise<void> {
     return Promise.resolve()
   }
 
+  /**
+   * Gets the job logs.
+   * 
+   * If the job has multiple containers, this aggregates the logs from them all.
+   */
   public logs(): Promise<string> {
     return Promise.resolve("skipped logs")
   }
+
+  /**
+   * Specifies a Runnable that consists of one or more containers. The
+   * image argument specifies the primary container, which determines
+   * how long the job runs and whether it is considered successful.
+   * By default, the primary container is simply
+   * executed (via its default entry point). Set the Job#primaryContainer#command
+   * property to run a specific command in the container instead.
+   * Other containers, specified via Job#sidecarContainers, are automatically
+   * terminated after the primary container completes.
+   * 
+   * (Note: This is equivalent to `new Job(...)`. It is provided so that
+   * script authors have the option of a consistent style for creating
+   * and composing jobs and groups.)
+   * 
+   * @param name The name of the job
+   * @param image The OCI image reference for the primary container
+   * @param event The event that triggered the job
+   */
+  public static container(
+    name: string,
+    image: string,
+    event: Event
+  ): Job {
+    return new Job(name, image, event)
+  }
+
+  /**
+   * Specifies a Runnable that consists of sub-Runnables (such as jobs
```

Comment:
Another non-blocking nit about "composed of language" possibly reading better. Another dealer's choice.